src/gcodeExport.cpp

```diff
@@ -889,15 +889,23 @@ void GCodeExport::writeJerk(double jerk)
 {
     if (current_jerk != jerk)
     {
-        if (getFlavor() == EGCodeFlavor::REPETIER)
+        if (getFlavor() == EGCodeFlavor::REPRAP)
         {
-            *output_stream << "M207 X";
+            const double jerkPerMin = jerk * 60;
+            *output_stream << "M566 X" << PrecisionedDouble{2, jerkPerMin} << " Y" << PrecisionedDouble{2, jerkPerMin} << new_line;
         }
         else
         {
-            *output_stream << "M205 X";
+            if (getFlavor() == EGCodeFlavor::REPETIER)
+            {
+                *output_stream << "M207 X";
+            }
+            else
+            {
+                *output_stream << "M205 X";
+            }
+            *output_stream << PrecisionedDouble{2, jerk} << new_line;
```

Comment:
It might be more readable and efficient to use an actual switch statement here on getFlavor(), to be either REPRAP, REPETIER or default. Otherwise I think the code itself is a fine fix (but pull requests are on halt for a few weeks on CuraEngine due to internal talks going on).